charts/rocketchat/values.yaml

```diff
@@ -138,6 +138,12 @@ mongodb:
 
   volumePermissions: { enabled: false }
 
+  podSecurityContext:
+    enabled: false
+
+  containerSecurityContext:
+    enabled: false
+
```

Comment:
Does this work well on OpenShift?

Reply:
Yes, it does. By default, the mongodb Helm Chart has the Pod and Container security context enabled. You can disable them on Openshift since it'll be generated automatically (I guess it's like an inheritance from OKD policy?)